# DMS information for selecting an action

string id
string behavior_id
string motivation
string behaviour_type
string emotion
float32 value